02Lesson/01Homework/css/style.css

```diff
@@ -0,0 +1,68 @@
+body
+  {
+  text-align: center;
+  margin: auto;
+  padding: 0;
+  max-width: 700px;
+  background-image: url("../images/orange_black_bg.jpg");
+  background-size: cover;
+  color: white;
+  font-size: 21px;
+  text-shadow:
+    -1px -1px 0 #000,
+    1px -1px 0 #000,
+    -1px 1px 0 #000,
+    1px 1px 0 #000;
+  }
+
+header
+  {
+  font-size: 30px;
+  }
+
+#main ul, #main li
+  {
+  text-align: center;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  }
+
+#nav {
+  text-align: center;
+  margin: 0;
+  padding: 0;
+  max-width: 128px;
+  }
+
+ul {
+    position: fixed;
+    top: 16px;
+    left: 8px;
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    width: 128px;
+    background-color: red;
+    }
+
+li a {
+    /* Default look of list items and links when not hovered over. */
+    display: block;
+    border-bottom: 1px solid black;
+    color: white;
+    padding: 8px 16px;
+    text-decoration: none;
+    }
+li a:hover {
+    /* The button is being hovered over. */
+    background-color: brown;
+    color: red;
+    opacity: 1;
+    }
+
+li a.active {
+    /* The button is being pressed down. */
+    background-color: red;
+    color: maroon;
+    }
```

Comment:
Nice use of comments! Try to use consistent indentation and positioning of brackets:
- Opening bracket always on the same line as selector
- Closing bracket not indented (Same indentation level as selector)